rst/telemetry.rst

```diff
@@ -32,22 +37,24 @@ Next, look for an assets reference in the match relationships object, which poin
     ...
   }
 
-Once you have located this ID, you now have to search for the object with that ID in the included array. The full object found there will provide you a link to the telemetry data, and look something like this::
+Once you have located this ID, you can search for the telemetry object which has that ID in the included array. The full object found there will provide you with a link to the telemetry file, and look something like this::
 
   {
     "type": "asset",
     "id": "1ad97f85-cf9b-11e7-b84e-0a586460f004",
     "attributes": {
-      "URL": "https://telemetry-cdn.pubg.com/pc-krjp/2018/01/01/0/0/1ad97f85-cf9b-11e7-b84e-0a586460f004-telemetry.json", //Note this link will not work

```

Comment:
Are you sure you don't want this? I had people email me back in the day because this link didn't work. 😀